primer/brand
# Bento Component

## Summary

Creation of the Bento component which will act as a presentational component. Most notably it will be used soon on the Universe page.

## List of notable changes:

- **added** Bento component

## What should reviewers focus on?

- The design and functionality works as expected based on the Figma component.

## Steps to test:

1. Open the Bento component in CI-deployed preview environment
2. Go to Bento and Bento.Item stories in Storybook
3. Verify that they behave as described in the following issue.

## Supporting resources (related issues, external links, etc):

- [API Exploration](https://www.figma.com/file/gwcGUHN5MRHWFH5I7l2Z75/Primer-Brand-Bento-API-exploration?type=whiteboard&node-id=0-1&t=mOh4HxcxQowQF17Y-0)
- [Expected Variants](https://www.figma.com/file/BsVDcDotz7pyQPR6L0HUIb/Bento?type=design&node-id=79-468&mode=design)
- [Figma Component Loom](https://www.loom.com/share/cf7c2fb3f1e54f89a9e7f7ebe88a9cf5)

## Contributor checklist:


## Reviewer checklist:


<img width="1131" alt="image" src="https://github.com/primer/brand/assets/31220082/26388b84-d9a3-4cfd-87e7-61c9cef02a89">
